CHANGELOG.md

```diff
@@ -18,10 +18,15 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
 
+- Added support for displaying a configurable management API URL for the user
+
 #### Patch
 
 #### Deployment Migration Notes
 
+- New environment variables:
+  - `EDC_UI_SHOWN_MANAGEMENT_API_URL_IN_DASHBOARD`
```

Comment:
- New **optional** environment variable: `EDC_UI...` as override for shown Management API URL in the dashboard